### ISSUE TITLE
upgrade icu4j

### ISSUE THREAD

**Issue by dankurka** [gwtproject/gwt#8703]
Originally reported on Google Code with ID 8737

```
(Reposting an internal Google issue: b/15277036)

gwt is using an older version of icu4j than the rest of Google.

Specifically, the generation of Japanese date patterns are different (Google version
is "correct").

With "old" icu used by GWT:
"EEE MMMM d" -> M月d日EEE
With "new" icu used in Google:
"EEE MMMM d" -> M月d日(EEE)  (mind the EEE in parenthesis)

The last result is correct according to the bug.

Would it be possible to upgrade icu4j version used in gwt?
Current version in GWT is 50.1.1 
Current version in Google is 53.1.
```

Reported by `skybrian@google.com` on 2014-05-27 21:07:26

**Comment by tbroyer** [comment]
Tentatively assigning to 2.9 milestone, as CLDR data is years behind and would really need to be updated (so ICU4J could possibly be updated at the same time); see #9557

**Comment by vegegoku** [comment]
I have the icu4j update ready, i am only waiting for the following PRs to be merged
https://gwt-review.googlesource.com/c/gwt/+/21400
 and 
https://github.com/gwtproject/tools/pull/15

**Comment by pethers** [comment]
is this fixed in latest 64.2 (https://mvnrepository.com/artifact/com.ibm.icu/icu4j/64.2) ?

**Comment by niloc132** [comment]
Looks like these were already merged.
